my.css

```diff
@@ -352,3 +352,172 @@ body {
     justify-content: center;
   }
 }
+
+/* Вторая секция */
+.bagscluth__content {
+  display: flex;
+  max-width: 1150px;
+  width: 100%;
+  margin: 0 auto;
+}
+.bagscluth {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  max-height: 900px;
+  display: flex;
+  background-color: #fae8e4;
+}
+.bagscluth__img {
+  width: 60%;
+  height: 844px;
+  display: flex;
+  align-items: center;
+}
+.bags-img {
+  position: absolute;
+  display: block;
+  width: 340px;
+  height: 400px;
+  padding-left: 10px;
+
+}
+.bags-img1 {
+  z-index: 1;
+  width: 540px;
+  height: 740px;
+  padding-left: 80px;
+}
+
+.bagclach__iner {
+  width: 45%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+
+}
+.card {
+  width: 488px;
+  padding: 30px 55px 45px 55px; 
+  margin: 0 auto;
+  top: 200px;
+  display: flex;
+  flex-flow: column;
+  background-color: #fff;
+  border: 1px solid #fff;
+  box-shadow: 29px 30px 22px 0px rgba(0,0,0,0.07);
+  box-sizing: border-box;
+
+}
+.card__header {
+  padding: 15px 15px;
+  margin: 0 auto;
+}
+.card__subheading {
+  margin: 0 auto;
+  padding-top: 10px;
+  
+}
+
+.card__text {
+ padding-top: 15px;
+ padding-bottom: 25px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)